tsconfig.json

```diff
@@ -2,10 +2,25 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "react-native-screens": ["./src"],
-      "react-native-screens/native-stack": ["./src/native-stack/index.tsx"],
+      "react-native-screens": ["src"],
+      "react-native-screens/native-stack": ["src/native-stack/index.tsx"],
       "react-native-screens/reanimated": ["src/reanimated/index.tsx"],
-      "react-native-screens/gesture-handler": ["src/gesture-handler/index.tsx"]
+      "react-native-screens/gesture-handler": ["src/gesture-handler/index.tsx"],
+      "@react-navigation/bottom-tabs": [
+        "link:../react-navigation/packages/bottom-tabs/src"
+      ],
+      "@react-navigation/core": ["link:../react-navigation/packages/core/"],
+      "@react-navigation/elements": [
+        "link:../react-navigation/packages/elements/src"
+      ],
+      "@react-navigation/native": ["link:../react-navigation/packages/native/"],
+      "@react-navigation/native-stack": [
+        "react-navigation/packages/native-stack/src"
```

Comment:
Only thing that bothers me is why don't we use `link: ...` here as everywhere else?

Reply:
seems not necessary at all, removed it: https://github.com/software-mansion/react-native-screens/pull/2174/commits/019d0fe79e5d1ceab2ab0bf854cc76c93d9314b6